code/controllers/subsystem/persistence.dm

```diff
@@ -259,6 +260,15 @@ SUBSYSTEM_DEF(persistence)
 	if(CONFIG_GET(flag/use_antag_rep))
 		CollectAntagReputation()
 	SaveRandomizedRecipes()
+	SavePanicBunker()
+
+/datum/controller/subsystem/persistence/proc/LoadPanicBunker()
+	var/bunker_path = file("data/bunker_passthrough.json")
+	if(fexists(bunker_path))
+		GLOB.bunker_passthrough = json_decode(file2text(bunker_path))
+		for(var/ckey in GLOB.bunker_passthrough)
+			if(daysSince(GLOB.bunker_passthrough[ckey]) >= CONFIG_GET(number/max_bunker_days))
```

Comment:
Uh, actually, you should use a variable with CONFIG_GET(number/max_bunker_days) as value instead of running the CONFIG_GET every loop.

Reply:
this is run exactly once, when the subsystem initializes